### DISCUSSION TITLE
Question about equations from lecture in 02_measurements-vectors

### DISCUSSION THREAD

**Discussion by AjeethUConn** [cooperrc/advanced-dynamics#25] [General]
I had a couple confusions when looking back through 02_measurements-vectors, and I just wanted to make sure I understood the math shown. These questions are in reference to the example problem from the video.

1. I understand the math that Prof. Cooper did to get the answers, but I'm not sure I know what the reference is for the vectors r_B/0 and r_C/0. Which directions are i, j, and k? Shouldn't $r_{B/O}$ be $0i + 3sin(75)j + 3cos(75)k$, and $r_{C/O}$ be $4i + 0j + 2k$?
2. What do the directions mean by "components parallel to edges" for (a)? Does that refer to the edges of the plate in the X direction, Y direction, or Z direction?

**Comment by AjeethUConn** [comment]
I'll add a picture of the problem from that lecture, which might help with answering my question:
![Question about first Dynamics Lecture ](https://github.com/cooperrc/advanced-dynamics/assets/115102528/6f0c2d4a-7aef-4920-971d-f7ce1b76ac74)

**Reply by cooperrc** [reply]
Looks like you're correct for labels of $r_{C/O}$ and $r_{B/O}$
